src/jvm/test/org/jetbrains/kotlinx/lincheck_test/util/Playground.kt

```diff
@@ -0,0 +1,110 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.kotlinx.lincheck.transformation
+
+import net.bytebuddy.dynamic.loading.ByteArrayClassLoader
+import org.objectweb.asm.ClassReader
+import org.objectweb.asm.ClassWriter
+import org.objectweb.asm.Opcodes
+import org.objectweb.asm.Type
+import org.objectweb.asm.commons.GeneratorAdapter
+import org.objectweb.asm.commons.InstructionAdapter.OBJECT_TYPE
+import org.objectweb.asm.commons.Method
+import org.objectweb.asm.util.TraceClassVisitor
+import java.io.PrintWriter
+import java.io.StringWriter
+import java.lang.invoke.MethodHandles
+
+fun renderBytecode(className: String = "GeneratedClass", returnType: String = "void", functionContent: GeneratorAdapter.() -> Unit): String {
+    val bytecode = generateClassBytecode(className, returnType, functionContent)
+
+    // Convert the bytecode to a readable format
+    val cr = ClassReader(bytecode)
+    val sw = StringWriter()
+    val pw = PrintWriter(sw)
+    cr.accept(TraceClassVisitor(pw), 0)
```

Comment:
Should we also give an option to apply Lincheck bytecode transformation to the passed bytecode?
I believe it would be helpful.